java/maven/src/org/netbeans/modules/maven/execute/CommandLineOutputHandler.java

```diff
@@ -294,13 +294,6 @@ private String readLine() throws IOException {
                         nextLine = line.substring(execEventIdx);
                         line = line.substring(0, execEventIdx);
                     }                   
-                    if (line.startsWith("[INFO] Final Memory:")) { //NOI18N
```

Comment:
@cstamas pretty sure this is a m2 relict. I do remember the line but since it isn't [anywhere](https://github.com/apache/maven/blob/69980eeafcb1c0371e48df89953390b6d939d61a/its/core-it-support/maven-it-sample-archetype/src/main/resources/archetype-resources/out.txt#L743) i suppose it was from m2 times?

Reply:
Yes, this is not logged anymore